accepted/0000-yarn-tasks.md

```diff
@@ -0,0 +1,103 @@
+- Start Date: 2018-11-21
+- RFC PR:
+- Yarn Issue: https://github.com/yarnpkg/yarn/issues/5773
+
+# Summary
+
+Yarn CLI command `yarn run` will be able to orchestrate various npm scripts, allowing them to run in parallel by default and matching any glob patterns.
+
+# Motivation
+
+Many build processes involve Grunt, Gulp, or other task runners that have great use cases but also require separate plugins, configuration files, and understanding of the API. `yarn run` would remove the need for installing these often global dpendencies and lower maintenance costs by providing a simpler interface defined in package.json. This would also cut down on the script run times and would be useful with workspaces as mentioned in https://github.com/yarnpkg/yarn/issues/6689.
+
+Feature Requirements:
+
+- Be able to run multiple commands in parallel across platform
+- Show and differentiate console output for each command

```

Comment:
This one is key. The hardest part of scripts ran in parallel is debugging the log later. "Which script generated the error and when?"